### PR TITLE
Tutorial

### PR DESCRIPTION
**Pull request by mortonjt**
This fixes the CF tutorial files

### CODE REVIEW

**Comment by mortonjt** [comment]
@cuttlefishh this fixes issue with the CF tutorial that you noticed.  Thanks for catching!

**Comment by mortonjt** [comment]
This is ready for review / merge please

File: ipynb/cfstudy/cfstudy-qiime2-tutorial.ipynb [hunk 1]
```diff
@@ -4,40 +4,19 @@
...
-    "!qiime tools import \\\n",
-    "    --input-path otu_table.biom \\\n",
-    "    --output-path cfstudy.biom.qza \\\n",
-    "    --type FeatureTable[Frequency]\n",
-    "\n",
-    "!qiime tools import \\\n",
-    "    --input-path cfstudy_taxonomy.txt \\\n",
-    "    --output-path cfstudy_taxonomy.qza \\\n",
-    "    --type FeatureData[Taxonomy]"
+    "In this study, there were 18 patients with cystic fibrosis.  The hypothesis was that there were two main communities at play in the CF lung.  One of these communities thrives at low pH, and the other community thrives at high pH.  To test this, sputum samples were divided among 8 tubes, and each of the tubes was perturbed with a different pH.  Here we will calculate balances, and test how these balances change with respect to pH, using linear mixed effects models."
```
Comment (antgonza) [inline ipynb/cfstudy/cfstudy-qiime2-tutorial.ipynb]: two main communities? I guess bacterial, right? when you say community you mean families/classes/orders or what? My guess is what ... 😄

**Comment by mortonjt** [comment]
The qza is sufficient - we can cut out the biom tables

**Comment by mortonjt** [comment]
Thanks!

**Pull request merged**